charts/linkerd-control-plane/values.yaml

```diff
@@ -288,10 +288,40 @@ identity:
       keyPEM: |
 
 
-# -|- CPU, Memory and Ephemeral Storage resources required by the identity controller (see `proxy.resources` for sub-fields)
-#identityResources:
-# -|- CPU, Memory and Ephemeral Storage resources required by proxy injected into identity pod (see `proxy.resources` for sub-fields)
-#identityProxyResources:
+# -|- CPU, Memory and Ephemeral Storage resources required by the identity controller
+identityResources:
+  cpu:
+    # -- Maximum amount of CPU units that the identity controller can use
+    limit: ""
+    # -- Amount of CPU units that the identity controller requests
+    request: ""
+  memory:
+    # -- Maximum amount of memory that the identity controller can use
+    limit: ""
+    # -- Amount of memory that the identity controller requests
+    request: ""
+  ephemeral-storage:
+      # -- Maximum amount of ephemeral storage that the identity controller can use
+      limit: ""
+      # -- Amount of ephemeral storage that the identity controller requests
+      request: ""
+# -|- CPU, Memory and Ephemeral Storage resources required by proxy injected into identity pod
+identityProxyResources:
```

Comment:
we don't have to block this PR but I don't understand why these configurations are separate from the other `identity.*` scope. we may want to revisit this prior to release (esp since the previous configurations were undocumented).

Reply:
agreed